docker-compose.yml

```
@@ -21,8 +21,8 @@ services:
             interval: 10s
             timeout: 10s
             retries: 3
-        # ports:
-        #    - "5432:5432"
+        ports:
```

This line should also be commented out
```suggestion
        # ports:
```